pandas/core/groupby/groupby.py

```diff
@@ -582,6 +582,15 @@ def wrapper(*args, **kwargs):
                kwargs_with_axis['axis'] is None:
                 kwargs_with_axis['axis'] = self.axis
 
+            if (name == 'hist' and
+                    kwargs.pop('equal_bins', False)):
```

Comment:
Can you get the same behavior without mutating the kwargs dict?

Reply:
Yes, I can pass `equal_bins` to `hist_series`. Do you also mean this for line 588? Just asking, because that might be a bit tricky to pipe through.

Reply:
Less concerned about 588 since you end up replacing that anyway

Reply:
Thanks. I ended up copying kwargs and mutating that.